src/Tribe/Admin_List.php

```diff
@@ -9,6 +9,7 @@
 }
 
 use Tribe__Events__Main as TEC;
+use Tribe__Events__Aggregator__Records as TEC_AR;
```

Comment:
```suggestion
use Tribe__Events__Aggregator__Records as Records;
```

https://github.com/moderntribe/the-events-calendar/pull/3395#discussion_r538862446

Reply:
Looks like this one is no longer required.

Reply:
Correct. I removed it.